dipy/workflows/tracking.py

```diff
@@ -23,8 +23,8 @@ class LocalFiberTrackingPAMFlow(Workflow):
     def get_short_name(cls):
         return 'track_local'
 
-    def _get_direction_getter(self, strategy_name, pam, pmf_threshold=0.1,
-                              max_angle=30.):
+    def _get_direction_getter(self, strategy_name, pam, pmf_threshold,
+                              max_angle):
```

Comment:
it seems I forgot to add all parameters to the docstring. Can you add them (`pam`, `pmf_threshold`, `max_angle`)?

Reply:
done.